### DISCUSSION TITLE
Feature request: Sync with Remember the Milk

### DISCUSSION THREAD

**Discussion by justyns** [bergercookie/syncall#122] [Ideas]
Thanks for making this tool.  I'm new to taskwarrior, but have been trying syncall out to sync tasks from nextcloud and google tasks.  I'd also like to use it for Remember The Milk since I have some shared lists there with my wife.

This issue is a feature request for supporting syncing with RTM.  It has a free api available:
https://www.rememberthemilk.com/services/api/

There are a few python libraries for it as well, but it doesn't look like they've been updated recently.

**Comment by bergercookie** [comment]
Hi @justyns,

Thanks for the kind words.
Noted. I may have a dig at it but I cannot guarrantee either a date or a completed result. If you do want to have a go, you can use PRs like the following as a template to the types of changes that need to happen (https://github.com/bergercookie/syncall/pull/63, https://github.com/bergercookie/syncall/pull/79). Basically you need to create a `SyncSide` class for Remember the Milk to interact with that software + 2 conversion utilities TW -> RTM, RTM -> TW and finally a rememberthemilk_tw_sync.py script that brings it all together. The rest should be taken care of automatically by the [item_synchronizer](https://github.com/bergercookie/item_synchronizer) and by the Aggregator class

**Comment by bergercookie** [comment]
6 monthly update :)

I don't have much time working on this but if this is still something of interest to people, please feel free to pick it up.

Moving this to a discussion
